vuepress/docs/next/tutorials/devops/caching-and-clustering.md

```diff
@@ -34,28 +35,32 @@ This tutorial reviews setting up a clustered Entando App Engine in `entando-de-a
 - The Entando deployment must use a Relational Database Management System (RDBMS) to organize data in a table structure. Clustered instances will not work correctly with in-memory databases.
 
 ### Creating a Clustered App Instance
-1. Create an Entando deployment via the Helm template or edit an existing deployment YAML file.
+1. Create an Entando deployment via the operator config file or edit an existing deployment YAML file.
 2. Scale your Entando server application:
 
 ``` bash
-kubectl scale deployment quickstart-server-deployment -n entando --replicas=2
+kubectl scale deployment quickstart-deployment -n entando --replicas=2
 ```
 
-3. Run `kubectl get pods -n [your namespace]` to view the pods in your deployment.
-4. You should have two `quickstart-server-deployment` pods in your namespace.
-5. The logs of the `quickstart-server-deployment` in either pod contains logging information related to different instances joining the cluster and how data is balanced between instances. See the screenshot below for an example (actual logs will vary).
+
+3. Run `kubectl get pods -n YOUR-NAMESPACE` to view the pods in your deployment.
+4. You should have two `quickstart-deployment` pods in your namespace.
+
+5. Look in the logs of the `quickstart-deployment` in either pod and you will see logging information related to different instances joining the cluster and balancing the data between the instances. See the screenshot for an example. Your actual logs will vary.
+
 
 ![Clustered Logs](./img/clustered-logs.png)
 
 ### Validating Clustered Instances
 This is an advanced exercise and not required or recommended for most deployment scenarios. The steps below validate that the clustered instances are working in your environment and that you have created a high availability deployment.
 
-1. [Create a clustered instance](#creating-a-clustered-app-instance) or have an existing clustered Entando App instance available for testing.
-2. Get the URL for your `entando-de-app` with `kubectl get ingress -n [your namespace]`.
+
+
+1. Complete the [creating a clustered instance tutorial](#creating-a-clustered-app-instance) above or have an existing clustered Entando App instance available for testing.
```

Comment:
capitalize tutorial like discussed?

Reply:
It's not a separate page or title, just a section above